iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensator.java

```diff
@@ -133,6 +135,28 @@ default OptionalInt findSectionCount() {
         return OptionalInt.of(getSectionCount());
     }
 
+    /**
+     * Get the solved count of sections in service. This count represent the result count after a calculation (load flow).
+     * <p>
+     * It is expected to be greater than one and lesser than or equal to the maximum section count.
+     * It can be null if no calculation has been launched on the network.
+     * <p>
+     * Depends on the working variant.
+     * @see VariantManager
+     */
+    Integer getSolvedSectionCount();
+
+    /**
+     * Get the solved count of sections in service if it is defined.
+     * Otherwise, get an empty optional.
+     * <p>
+     * Depends on the working variant.
+     * @see VariantManager
+     */
+    default Optional<Integer> findSolvedSectionCount() {
```

Comment:
What is the reason to use `Optional<Integer>` instead of `OptionalInt` as for `findSectionCount()`?
I know `sectionCount` is an int and `solvedSectionCount` is an Integer. But in both cases, when `isEmpty()` is false on the result, you have a non-null value.